docs/run-command.md

```diff
@@ -8,12 +8,29 @@ of running scripts in an Azure Linux VM also apply here.
 
 ## Regardless of the Kubernetes control plane status
 
-When executing `run-command` without passing through Kubernetes, we need to provide the information of the node (VMSS instance) where we want to run the given command. To retrieve such information, we can use the [`config import`](./config.md#importing-configuration) command. Once we got it, we can select the node we want to use and all the subsequent `run-command` commands will be executed on that node:
+When executing `run-command` without passing through Kubernetes, we need to provide the information of the node (VMSS instance) where we want to run the given command. To retrieve such information, we can use the [`config import`](./config.md#importing-configuration) command. Once we got it, we can select the node we want to use and all the subsequent `run-command` commands will be executed on that node.
+To try it start by cleaning the current configuration (if any)
 
 ```bash
-# Import the nodes information with the cluster information
-$ kubectl aks config import --subscription mySubID --resource-group myRG --cluster-name myCluster
-$ kubectl aks config show
+kubectl aks config unset-all
+```
+
+Import the nodes information with the cluster information:
+
+```bash
+kubectl aks config import --subscription $mySubID --resource-group $myRG --cluster-name $myCluster
+```
+
+In case we want to print the imported information, we can use the `show` command:
+
+```bash
+kubectl aks config show
+```
```

Comment:
If you check, we always execute `config show` after a `config import`. I started thinking that we should automatically print the imported data in the `config import` (maybe with a `--quiet` flag). Also for the documentation, it makes things more consistent. For instance:

Import node info:
```bash
kubectl aks config import
```

Run command against one of the nodes (Folks will take the node name from the previous output):

```bash
kubectk aks run-command --node <myNode>
```

WDYT?

Reply:
https://github.com/Azure/kubectl-aks/issues/60